spec/basket_spec.rb

```diff
@@ -2,19 +2,35 @@
 
 RSpec.describe Basket do
 
-  it "creates basket" do
-    expect Basket.new.is_a? Basket
+  let(:product1) { Product.new(name: "Football", price: 79) }
+  let(:product2) { Product.new(name: "Basketball", price: 60) }
+  let(:basket1) { Basket.new([product1, product2]) }
+
+  it "calculates products sum correctly" do
```

Comment:
Agreed on creating context/describe for those two.
As mentioned before, not sure what you mean by testing them seperately (see my comment for line 24).